### PR TITLE
[Sprint 47] XD-2855 [master + backport] Update config to enable basic auth role

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- Add config to enable roles for basic authentication
  - Update the documentation

### CODE REVIEW

File: src/docs/asciidoc/Application-Configuration.asciidoc [hunk 1]
```diff
@@ -397,6 +397,7 @@ security:
   basic:
     enabled: true                                                     # <2>
     realm: SpringXD                                                   # <3>
+    role: ADMIN, VIEW, CREATE                                         # <4>
```
Comment (dturanski) [inline src/docs/asciidoc/Application-Configuration.asciidoc]: This doesn't seem to match the layout in servers.yml:

```
#security:
#  basic:
#    enabled: true # false to disable security settings (default)
#    realm: SpringXD
#  user: # valid only if security.basic.enabled=true
#    name: yourAdminUsername
#    password: yourAdminPassword
#    role: ADMIN, VIEW, CREATE
```

Comment (ilayaperumalg) [inline src/docs/asciidoc/Application-Configuration.asciidoc]: Yeah, sorry. The servers.yml entry is the correct one. I will fix the documentation.

**Comment by ilayaperumalg** [comment]
Updated the PR to fix the doc issue.

**Comment by dturanski** [comment]
LGTM; merged.

**Pull request closed without merging**